convert2rhel/actions/pre_ponr_changes/subscription.py

```diff
@@ -195,7 +195,15 @@ def run(self):
                     return
                 raise
 
-            logger.warning("No rhsm credentials given to subscribe the system. Skipping the subscription step.")
+            if subscription._is_registered() and not subscription.is_sca_enabled():
+                self.set_result(
+                    level="ERROR",
+                    id="SYSTEM_REGISTERED_WITHOUT_SCA",
+                    title="Registered with RHSM but without SCA enabled",
+                    description="This system has been registered with Red Hat Subscription Manager but Simple Content Access is not enabled.",
+                    remediation="To resolve this error please enable Simple Content Access and run the conversion again.",
```

Comment:
How is the recommendation when it comes to writing SCA vs Simple Content Access? Can we use SCA in titles or should we use the full name? I'll ping Miriam

Reply:
Both Pino and Miriam says it is spelled out first, then abbreviated, like `Simple Content Access (SCA)`. So I would change to something like this

```suggestion
                    id="SYSTEM_REGISTERED_WITHOUT_SCA",
                    title="Registered with RHSM but without Simple Content Access (SCA) enabled",
                    description="This system has been registered with Red Hat Subscription Manager but Simple Content Access is not enabled.",
                    remediation="To resolve this error please enable Simple Content Access and run the conversion again.",
```